UI-extra/index.html

```diff
@@ -21,21 +21,21 @@
                         </div>
                     </button>
                 </div>
-                <div class="flex-item">
-                    <a class="site-header-icons">
+                <div class="flex-item padding-sides-m">
+                    <a href="www.logo.com" >
                         <div class="logo">
                             <img class="header-icons" src="./icons/social-fb.png">
                         </div>
                     </a>
                 </div>
                 <div  class="flex-static padding-sides-m">
-                    <a class="site-header-icons">
+                    <a href="www.account.com" class="site-header-icons">
                         <div>
                             <img class="header-icons" src="./icons/social-fb.png">
                         </div>
                     </a>
                 </div>
-                <div class="flex-static">
+                <div class="flex-static padding-sides-m">
```

Comment:
Right now the header buttons/anchors have different size, we'll have to make sure the size is consistent.

![size](https://cloud.githubusercontent.com/assets/4543395/17197415/76c86ecc-5421-11e6-9d38-12d0e69118d5.gif)